### DISCUSSION TITLE
Extension that loads files from project directory/subdirectory?

### DISCUSSION THREAD

**Discussion by valkalyne** [TurboWarp/extensions#1073] [Ideas]
meowdy. im usin the video extension for animated cutscenes in my game thingy. However, i want the game to be completely runnable offline and i think storin all the cutscenes in a list is not very memory or speed efficient... ive figured if i stored all of the cutscenes in a folder i could load them at runtime, but i dont think thats possible. would someone be Oh So Kind as to program this, or tell me a way that i can do this. thank u very muche :]

**Comment by BlueDome77** [comment]
You could try the Zip extension
